### DISCUSSION TITLE
Get position of objects for each frame

### DISCUSSION THREAD

**Discussion by timotoots** [blakeblackshear/frigate#5325] [Ask A Question]
I would like to create more advanced automated editing based on events. Looking through the features it seems it is not possible at the moment to get info about each frame of the clip. I would be interested in the same things as in debug view, but in just JSON format.

For example I would like to try editing the recorded clip by the location of the object (eg. person). So you could zoom in on action not see wide view. And follow the action through the frames.

Is there a simple DEBUG flag somewhere I could turn on and just parse the logs for a hacky start? Or even better, send live detections to MQTT for external processing. 

Thanks!

**Comment by NickM-27** [comment]
Not currently but something like this metadata will potentially be implemented in the next version

**Reply by timotoots** [reply]
Thanks for answer, NickM! I think it would be useful!
